docs/Account sign in (Account Protection) - Approved and rejected sign in flows.md

```diff
@@ -0,0 +1,88 @@
+# Microsoft Dynamics 365 Fraud Protection - API examples
+## Account sign in (Account Protection APIs) - Approved and rejected sign in flows
+
+After sending Dynamics 365 Fraud Protection a sign in event, use your merchant rule decision returned by Dynamics 365 Fraud Protection to either continue with or stop your sign in workflow.
+
+## Helpful links
+- [Calling Dynamics 365 Fraud Protection](./Authenticate&#32;and&#32;call&#32;Fraud&#32;Protection.md)
+- [SignIn - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpPost)
+- [Sample site - Sign in](../src/Web/Controllers/AccountController.cs) (see SignIn POST method)
+- [Sample site - Dynamics 365 Fraud Protection service](../src/Infrastructure/Services/FraudProtectionService.cs) (see PostSignInAP method)
+
+## Sign in flow
+The sign in event flow consists of:
+1. Sign in event
+
+**NOTES**
+- - Set the 'assessmentType' field based on if you plan to use the Dynamics 365 Fraud Protection risk score:
+  - Pass 'Evaluate' if you do NOT plan to use the risk score and are still evaluating Dynamics 365 Fraud Protection against your existing fraud solution.
+  - Pass 'Protect' if you plan to use the risk score . This creates more accurate reports when we can distinguish between your 'Evaluate' and 'Protect' API calls.
+- In the sample site, an existing user's username is set to their email address which is also tied to how they log in to the sample site. Furthermore, the sample site ensures that no two customers have the same email address. This ensures usernames will be unique for the sample site, as well as unique when sent to Dynamics 365 Fraud Protection. Decide what format to use for existing user IDs. It does not have to be an email address, but you should be careful not to duplicate user IDs.
+
+## Required data
+- Login ID
+- Customer local date
+- Merchant local date
+- Assessment type
+- Session ID
+- Username
+- Password hash
```

Comment:
Regardless of if it's required or not, I think it would be prudent for us to put a small blurb on this page about what the password hash is.  I know the first time I saw it in the API I didn't understand what I (as the merchant) was supposed to send.

Reply:
added a note about password hash in the sample payload